run/test/specs/group_disappearing_messages_link.spec.ts

```diff
@@ -9,23 +10,21 @@ import { setDisappearingMessage } from './utils/set_disappearing_messages';
 iosIt('Disappearing link to group', disappearingLinkMessageGroup);
 androidIt('Disappearing link to group', disappearingLinkMessageGroup);
 
-// bothPlatformsIt(
-//   "Send disappearing link to group",
-//   disappearingLinkMessageGroup
-// );
+const timerType = 'Disappear after send option';
+const time = DISAPPEARING_TIMES.THIRTY_SECONDS;
 
 async function disappearingLinkMessageGroup(platform: SupportedPlatformsType) {
   const { device1, device2, device3 } = await openAppThreeDevices(platform);
   const testGroupName = 'Test group';
-  const testLink = `https://type-level-typescript.com/objects-and-records`;
+  const testLink = `https://getsession.org/`;
```

Comment:
maybe we should use the stagenet one, if we have one publicly accessible?

Reply:
if we're talking about something like https://token-stg.getsession.org those sites are password protected and will not generate a preview on Session